components/cartlist-card/index.js

```diff
@@ -107,12 +107,23 @@ export const CartCard = (props) => {
           display: inline-block;
           background-color: ${colors.pink.dark};
           font-size: 16px;
+          text-align: center;
         }
 
         @media only screen and (max-width: 600px) {
           .cartcard-container {
             flex-direction: column;
           }
+          .cartcard-button{
+            display: flex;
+            flex-direction: row;
+            justify-content: space-evenly;
+          }
+          .cartcard-fnbutton{
+            display: flex;
+            flex-direction: row;
```

Comment:
NIT: When we use display: flex by default the flex-direction is row, so even if we do not specify flex-direction: row it will automatically consider it as row

Reply:
@sumitd94 I have make it correct now. Please check.